web/satellite/src/store/modules/projects.ts

```diff
@@ -98,7 +98,7 @@ export function makeProjectsModule(api: ProjectsApi): StoreModule<ProjectsState>
                 state.projects = state.projects.filter(proj => proj.id !== projectID);
 
                 if (state.selectedProject.id === projectID) {
-                    state.selectedProject = new Project('', '', '', '');
+                    state.selectedProject = new Project('', '', '', '', '');
```

Comment:
I guess that can be omitted because constructor has default parameters

Reply:
done